reactive/kotlinx-coroutines-rx2/src/RxAwait.kt

```diff
@@ -52,12 +53,23 @@ public suspend fun <T> MaybeSource<T>.await(): T? = (this as MaybeSource<T?>).aw
  * If the [Job] of the current coroutine is cancelled or completed while this suspending function is waiting, this function
  * immediately resumes with [CancellationException].
  */
-public suspend fun <T> MaybeSource<T>.awaitOrDefault(default: T): T = suspendCancellableCoroutine { cont ->
+public suspend fun <T : Any> MaybeSource<T>.awaitOrDefault(default: T): T = suspendCancellableCoroutine { cont ->
```

Comment:
Haven't we decide to leave them as extensions on `T : Any?` for the sake of everybody's convenience? With the changes in this file all the generic usages for these extensions will have to be updated.